src/util-radix-tree.c

```diff
@@ -1355,21 +1355,6 @@ static void SCRadixRemoveKey(uint8_t *key_stream, uint16_t key_bitlen,
     return;
 }
 
-/**
```

Comment:
separately, it would be good to remove the unused "Generic" support from the radix code

Reply:
> unused "Generic" support from the radix code

What is this ?

Reply:
the radix code can store 3 types of data: ip4, ip6 and "generic". But generic isn't used.

Reply:
Not sure I see it : for instance
`SCRadixAddKey` is only called by IPv4 and IPv6 functions